nat-lab/tests/test_fire_connecting_event.py

```diff
@@ -3,15 +3,14 @@
 from contextlib import AsyncExitStack
 from helpers import SetupParameters, setup_mesh_nodes
 from telio import AdapterType, State
-from utils import testing
 from utils.connection_tracker import ConnectionLimits
 from utils.connection_util import generate_connection_tracker_config, ConnectionTag
 from utils.ping import Ping
 
 
 @pytest.mark.asyncio
 @pytest.mark.long
-@pytest.mark.timeout(180 + 60)
+@pytest.mark.timeout(240)
```

Comment:
Can we parametrize all the different timeout values ? 
Changing the values later on would require only a couple of changes.

Reply:
Done!